aws/fedora-coreos/kubernetes/workers/variables.tf

```diff
@@ -108,26 +108,3 @@ variable "node_labels" {
   description = "List of initial node labels"
   default     = []
 }
-
-# Scoop variables
-
-variable "ami" {
```

Comment:
Aren't these important?

Reply:
@bendrucker These were not shown as merge conflicts either. Otherwise there's no way I can make these changes

Reply:
Let me review all the changes. Cannot really trust merge